core/src/tpu.rs

```diff
@@ -41,6 +44,12 @@ use {
 
 pub const DEFAULT_TPU_COALESCE_MS: u64 = 5;
 
+/// The default maximum number of batches in the queue that leaves the
+/// fetch stage.
+///
+/// 10k batches means up to 1.3M packets, roughly 1.6GB of memory
+pub const DEFAULT_TPU_MAX_QUEUED_BATCHES_UDP: usize = 10_000;
```

Comment:
This value gets used for lots of channels now. Should we rename it to `DEFAULT_MAX_QUEUED_BATCHES_PER_CHANNEL` and apply it everywhere?

Reply:
Yes, agree. Renamed and moved into bounded_streamer.rs